content/Our_sponsers.php

```diff
@@ -87,54 +87,77 @@ function collage() {
+			</tr>
+			<tr><!--5-->
+				<td><a target="_blank" href="http://www.tnb.com/">Thomas and Betts - A member of the ABB group</a></td> <!--Titanium 5,000-->
+				<td><a target="_blank" href="http://www.linear.com">Linear Technology</a></td> <!--Aluminum 2,500-->
+				<td><a target="_blank" href="https://www.riverviewbank.com/">Riverview Bank</a></td> <!--Wood 1,000-->
+				<td><a target="_blank" href="http://www.seasonsfinancialonline.com/">Seasons Financial</a></td> <!--Plastic 500-->
+				<td class="last"></td> <!--Zip Tie 100-->
+			</tr>
+			<tr><!--6-->
+				<td></td> <!--Titanium 5,000-->
+				<td>Lange Family</td> <!--Aluminum 2,500-->
+				<td><a target="_blank" href="http://www.deltamotion.com/index.php">Delta Computer Systems</a></td> <!--Wood 1,000-->
+				<td><a>Barsotti Family</a></td> <!--Plastic 500-->
+				<td class="last"></td> <!--Zip Tie 100-->
+			</tr>
+			<tr><!--7-->
 				<td></td> <!--Titanium 5,000-->
-				<td><a target="_blank" href="http://www.aosmd.com/">Jireh Semiconductor</a></td> <!--Aluminum 2,500-->
-				<td><a>Linda Calica</a></td> <!--Wood 1,000-->
-				<td><a>EO Technical</a></td> <!--Plastic 500-->
+				<td></td> <!--Aluminum 2,500-->
+				<td></td> <!--Wood 1,000-->
+				<td><a>Industrial Machine Services</a></td> <!--Plastic 500-->
 				<td class="last"></td> <!--Zip Tie 100-->
 			</tr>
-			<tr>
+			<tr><!--8-->
 				<td></td> <!--Titanium 5,000-->
-				<td><a target="_blank" href="http://www.linear.com">Linear Technology</a></td> <!--Aluminum 2,500-->
-				<td><a target="_blank" href="http://www.local1159.com/">Professional Firefighters Local 1159</a></td> <!--Wood 1,000-->
-				<td><a target="_blank" href="http://www.protechcomposites.com/">Protech Composites</a></td> <!--Plastic 500-->
+				<td></td> <!--Aluminum 2,500-->
+				<td></td> <!--Wood 1,000-->
+				<td><a target="_blank" href="http://www.gates.com/">Gates</a></td> <!--Plastic 500-->
 				<td class="last"></td> <!--Zip Tie 100-->
-			<tr>
+			</tr>
+			<tr><!--9-->
 				<td></td> <!--Titanium 5,000-->
-				<td><a target="_blank" href="http://www.k12.wa.us/">OSPI</a></td> <!--Aluminum 2,500-->
-				<td><a target="_blank" href="https://www.riverviewbank.com/">Riverview Bank</a></td> <!--Wood 1,000-->
-				<td><a target="_blank" href="http://retailimaging.com/">Retail Imaging Managment Group</a></td> <!--Plastic 500-->
+				<td></td> <!--Aluminum 2,500-->
+				<td></td> <!--Wood 1,000-->
+				<td><a target="_blank" href="http://www.protechcomposites.com/">Protech Composites</a></td> <!--Plastic 500-->
 				<td class="last"></td> <!--Zip Tie 100-->
 			</tr>
-			<tr>
+			<tr><!--10-->
 				<td></td> <!--Titanium 5,000-->
 				<td></td> <!--Aluminum 2,500-->
-				<td><a>TCMS</a></td> <!--Wood 1,000-->
-				<td><a target="_blank" href="http://www.seasonsfinancialonline.com/">Seasons Financial</a></td> <!--Plastic 500-->
+				<td></td> <!--Wood 1,000-->
+				<td><a>Gencom Northwest</a></td> <!--Plastic 500-->
```

Comment:
We discussed this and the <a> sets the color of the text and looking at existing ones they don't appear as a link without the href.